runtime/description.js

```diff
@@ -351,7 +351,8 @@ export class DescriptionFormatter {
         let storeValue = await this._formatStoreValue(token.handleName, token._store);
         if (!description) {
           // For singleton handle, if there is no real description (the type was used), use the plain value for description.
-          if (storeValue && !token._store.type.isCollection && !this.excludeValues) {
+          // TODO: should this look at type.getContainedType() (which includes references), or maybe just type.isEntity?
+          if (storeValue && !this.excludeValues && !token._store.type.isCollection) { // && !token._store.type.isBigCollection) {
```

Comment:
why did you comment the `&& !...isBigCollection`? i think here we want to return storeValue only for singletons (maybe just check .isEntity? not sure)

Reply:
Oops, left-over debugging hacks...